.clang-tidy

```diff
@@ -0,0 +1,9 @@
+# Settings file automatically used by clang-tidy
+#
+# See ./contrib/utilities/run_clang_tidy.sh for details
+
+# We disable performance-inefficient-string-concatenation because we don't care about "a"+to_string(5)+...
+
+Checks: "-*,cppcoreguidelines-pro-type-static-cast-downcast,google-readability-casting,modernize-*,-modernize-pass-by-value,-modernize-raw-string-literal,-modernize-use-auto,-modernize-use-override,-modernize-use-default-member-init,-modernize-use-transparent-functors,use-emplace,mpi-*,performance-*,-performance-inefficient-string-concatenation"
```

Comment:
Nice! I also thought about using this possibility, but just tried once and it didn't work out-of-the-box.

Reply:
the thing that is broken is that you can not have newlines in the string here. :(